src/js/angular/ttyg/services/ttyg-context.service.js

```diff
@@ -171,6 +179,36 @@ function TTYGContextService(EventEmitterService) {
         emit(TTYGEventName.AGENT_SELECTED, getSelectedAgent());
     };
 
+    /**
+     * @return {{[key: string]: ExplainResponseModel}} the cache of explain responses.
+     */
+    const getExplainResponseCache = () => {
```

Comment:
better remove this method and create one getExplainResponse(id) to encapsulate the implementation. See previous comments

Reply:
Done